include/usearch/usearch.hpp

```diff
@@ -412,7 +422,12 @@ class max_heap_gt {
  *
  */
 class mutex_t {
+#ifdef WINDOWS
```

Comment:
`#if defined(WINDOWS)`

Reply:
Why?

Reply:
Because that's our style everywhere else. It makes searching easier.